src/main/resources/translations.json

```
@@ -1,7 +1,51 @@
 {
   "de-DE": [
     {
-
+      "test": "Dies ist ein test %s",
+      "arenaIsFull": "Die Arena ist gerade besetzt, du wurdest in die Warteliste aufgenommen",
+      "addedToPlayerList": "Du wurdest der Liste hinzugefügt und hast eine Difficulty von: ",
+      "playerNotOP": "Du bist nicht dazu berechtigt",
+      "changedDiff1": "Deine Schwierigkeit wurde um 1 auf ",
+      "changedDiff2": " erhöht",
+      "diffAlreadyAt": "Deine Schwierigkeit ist schon bei ",
+      "diffAt": "Deine Schwierigkeit ist gerade bei ",
+      "notEnoughResources": "Du bist zu pleite um das zu bezahlen",
+      "playerDecided": "Du hast dich entschieden.",
+      "playerAlreadyDecided": "Du hast dich bereits entschieden.",
+      "startingMinigame": "Starte Minispiel...",
+      "droppingInvAt": "Dein Inventar wird an deinen Todesort  gedroppt (",
+      "droppingInvAt2": "Dein Inventar wird an deinem Todesort gedroppt",
+      "setDiffTo1": "Deine Schwierigkeit wurde auf ",
```

String templates should be used
```suggestion
      "setDiffTo1": "Deine Schwierigkeit wurde auf %s gesetzt",
```

---

i split it like that because there parts of the message in between these two and those parts have diffferent styles, therefore I have to put them in differen text components